File: Selenium/Project/activity5.py
#Getting colors
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.color import Color

driver = webdriver.Firefox()
driver.get("http://alchemy.hguy.co/crm")

driver.find_element(By.ID, "user_name").send_keys("admin")
driver.find_element(By.ID, "username_password").send_keys("pa$$w0rd")
driver.find_element(By.NAME, "Login").click()

toolbar = driver.find_element(By.ID,"toolbar")
toolbarColor = toolbar.value_of_css_property("color")

#convert rgba to hex
hexColor = Color.from_string(toolbarColor).hex
print("Color is :" + toolbarColor)
print("Hex code for color:" + hexColor)

driver.quit()

File: Selenium/Project/activity5.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.junit.jupiter.api.Assertions;
import org.openqa.selenium.support.Color;

//Getting colors

public class activity5 {
    public static void main(String[] args) throws InterruptedException {
        // Install driver
        WebDriverManager.firefoxdriver().setup();

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "NULL");
        WebDriver driver = new FirefoxDriver();

        driver.get("http://alchemy.hguy.co/crm");
        driver.findElement(By.id("user_name")).sendKeys("admin");
        driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
        driver.findElement(By.name("Login")).click();

        WebElement toolbar = driver.findElement(By.id("toolbar"));
        String toolbarColor = toolbar.getCssValue("color");

        // convert rgba to hex
        String hexColor = Color.fromString(toolbarColor).asHex();
        System.out.println("Color is :" + toolbarColor);
        System.out.println("Hex code for color:" + hexColor);

        driver.quit();
    }
}